From: LWJGL SunOS Builder
Date: Thu, 23 Oct 2026 00:00:00 +0000
Subject: Fix missing string.h include in stb_dxt.h for GCC 15

GCC 15 rejects implicit declaration of memcpy. Add the required include.

--- a/modules/lwjgl/stb/src/main/c/stb_dxt.h
+++ b/modules/lwjgl/stb/src/main/c/stb_dxt.h
@@ -84,6 +84,7 @@

 #include <stdlib.h>

+#include <string.h>
 #if !defined(STBD_FABS)
 #include <math.h>
 #endif
